src/Database/Adapter.php

```diff
@@ -142,12 +142,14 @@ abstract public function create(string $name): bool;
 
     /**
      * Check if database exists
+     * Optionally check if collection exists in database
      *
-     * @param string $name
+     * @param string $database database name
+     * @param string $collection (optional) collection name
      *
      * @return bool
      */
-    abstract public function exists(string $name): bool;
+    abstract public function exists(string $database, string $collection): bool;
```

Comment:
```suggestion
    abstract public function exists(string $database, string $collection = ''): bool;
```
If this is optional, let's give a default value to `$collection`

Reply:
We should rather default to `null`, that way we can differentiate from an actual empty string and is nullable.